ice.c

```diff
@@ -1856,9 +1863,8 @@ int janus_ice_setup_local(janus_ice_handle *handle, int offer, int audio, int vi
 				JANUS_LOG(LOG_ERR, "[%"SCNu64"] getnameinfo() failed: %s\n", handle->handle_id, gai_strerror(s));
 				continue;
 			}
-			/* Skip localhost and 0.0.0.0 */
-			if(!strcmp(host, "127.0.0.1") || !strcmp(host, "0.0.0.0")
-					|| !strcmp(host, "::1") || !strcmp(host, "::") || strchr(host, '%'))
+			/* Skip 0.0.0.0, :: and local scoped addresses  */
+			if(!strcmp(host, "0.0.0.0") || !strcmp(host, "::") || strchr(host, '%'))
```

Comment:
> That doesn't end up true for ::1; you probably want to use !strncmp(host, "::", 2)
> 

That should be taken cared of above, see the check for IFF_LOOPBACK.

> I think the strchr(host, '%') is attempting to match ipv6 automatic non-routable subnet addresses; it may be better to do !strncmp(host, "fe80:", 5) for that.

Both should work, yes, but the fe80 test look smore explicit, I'll add it.